features/step_definitions/common.js

```
@@ -0,0 +1,18 @@
+const { Given, Then } = require('@cucumber/cucumber');
+
+Given('Pending', function () {
+  // Write code here that turns the phrase above into concrete actions
+  return 'pending';
+});
+
+Then('Verify screenshot {string}', { timeout: 30000 }, async function (namespace) {
+  // selenium
+  try {
+      const screenshot = await this.seleniumDriver.takeScreenshot();
+      const screenshotPath = path.resolve(process.cwd(), 'coverage/' + namespace + '.png');
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)